core/src/test/java/io/undertow/server/handlers/RoutingHandlerTestCase.java

```diff
@@ -126,6 +126,12 @@ public void handleRequest(HttpServerExchange exchange) throws Exception {
                         exchange.writeAsync("posted foo");
                     }
                 })
+                .add(HttpMethodNames.POST, "/foo/{baz}", new HttpHandler() {
+                    @Override
+                    public void handleRequest(HttpServerExchange exchange) throws Exception {
+                        exchange.writeAsync("foo-path" + exchange.getQueryParameters().get("bar"));
```

Comment:
I believe it should be like in https://github.com/undertow-io/undertow/pull/799/files#diff-b365caf66586093f040e0a2915a095a4bf1f4ed87343a31998507406f358e827R132?
```suggestion
                        exchange.getResponseSender().send("foo-path" + exchange.getQueryParameters().get("bar"));
```